src/drivers/mct/cime_config/config_component.xml

```diff
@@ -2607,64 +2607,6 @@
     <desc>External script to be run after model completion</desc>
   </entry>
 
- <!-- Job Submission stuff -->
```

Comment:
Since both e3sm and cesm are using env_workflow.xml now, shouldn't we be able to continue to share these entries?

Reply:
yes, I started out with the idea that I would only change cesm but then decided that there was no reason not to change e3sm as well.